modules/dbparser/dbparser.c

```diff
@@ -218,10 +220,9 @@ log_db_parser_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *pat
 
   if (self->drop_unmatched && !matched)
     {
-      msg_debug("db-parser() failed to parse its input and drop-unmatched flag was specified",
-                evt_tag_str("input", input),
-                log_pipe_location_tag(&s->super),
```

Comment:
Is this tag accidentally removed?

Reply:
No it was intentional, there would be a redundancy. From now log_pipe_location_tag() would be printed in every case from the very first place: log_source_queue(), log_parser_queue(), log_rewrite_queue(), log_filter_queue().